packages/augur-core/src/contracts/para/ParaShareToken.sol

```diff
@@ -108,12 +108,12 @@ contract ParaShareToken is ITyped, Initializable, ERC1155, ReentrancyGuard {
     }
 
     function buyCompleteSetsInternal(IMarket _market, address _account, uint256 _amount) internal returns (bool) {
-        if (!isMarketInitialized(_market)) {
+        uint256 _numTicks = markets[address(_market)].numTicks;
+        if (_numTicks == 0) { // !isMarketInitialized(_market)
             initializeMarket(_market);
         }
 
         uint256 _numOutcomes = markets[address(_market)].numOutcomes;
-        uint256 _numTicks = markets[address(_market)].numTicks;
```

Comment:
If the market isn't initialized and this is called wouldn't numTicks be 0 then at this point? I think it needs to be set again in the numTicks == 0 conditional block.

Reply:
ARGH right!